numpy/core/defchararray.py

```diff
@@ -377,6 +399,11 @@ def capitalize(a):
     return _vec_string(a_arr, a_arr.dtype, 'capitalize')
 
 
+def _center_dispatcher(a, width, fillchar=None):
+    return (a,)
```

Comment:
This is now unused

Reply:
It appears in the call in line 422. Do you mean not tested?

Reply:
I reverted my later commit that made this unused. So now it definitely *is* used.